src/App.vue

```diff
@@ -914,8 +937,17 @@ export default class App extends Mixins(
 </script>
 
 <style lang="scss" scoped>
+// display drop-down menu on top of stepper and fee summary
+::v-deep .app-header {
+  z-index: 3;
```

Comment:
By default the SBC Header has a z-index of 2. What weird is that the user drop-down has a z-index of 8 but it was not honoured here.

I didn't want to change SBC Header for everyone (ie, update common components) because there are various uses of z-index in common components and I'm not familiar with how they all interact.